datastore/datastore.go

```diff
@@ -66,21 +87,27 @@ func (self ElasticDatastore) GetSubject(
 	message proto.Message) error {
 
 	id := services.MakeId(path.AsClientPath())
-	data, err := services.GetElasticRecord(
-		self.ctx, config_obj.OrgId, "datastore", id)
+
+	hits, _, err := services.QueryElasticRaw(self.ctx, config_obj.OrgId,
+		"transient", json.Format(get_datastore_doc_query, id))
```

Comment:
I would have expected this index name to have already been changed in a previous PR. Were there any others that were missed.